### PR TITLE
rename the test e2e-vsphere to e2e-vsphere-ovn to make the testing clearer 

### PR DESCRIPTION
**Pull request by JacobTanenbaum**
in the ovn and cluster-network-operator change the name of the e2e-vsphere test
to e2e-vsphere-ovn so it is clearer what network plugin is being tested

renaming the test name on master, release-4.7, and release-4.6 for consistency

### CODE REVIEW

File: ci-operator/jobs/openshift/cluster-network-operator/openshift-cluster-network-operator-master-presubmits.yaml [hunk 1]
```diff
@@ -1025,26 +1025,26 @@ presubmits:
     always_run: true
     branches:
     - master
-    cluster: vsphere
-    context: ci/prow/e2e-vsphere
+    cluster: api.ci
+    context: ci/prow/e2e-vsphere-ovn
```
Comment (danwinship) [inline ci-operator/jobs/openshift/cluster-network-operator/openshift-cluster-network-operator-master-presubmits.yaml:1029]: that `cluster` change looks wrong? autogenerated thing that needs to be un-autogenerated maybe?

**Review by danwinship — COMMENTED** [review]
The commit says "azure" but the patch is vsphere...

File: ci-operator/jobs/openshift/cluster-network-operator/openshift-cluster-network-operator-master-presubmits.yaml (hunk 1 above)
Comment (JacobTanenbaum) [reply ci-operator/jobs/openshift/cluster-network-operator/openshift-cluster-network-operator-master-presubmits.yaml:1029]: you are right there is nothing that scheduled the test for vsphere automatically. Changed it

**Comment by abhat** [comment]
Should we do this for release-4.6 and release-4.7 branches too? Right now only master will see this distinction.

**Review by stevekuznetsov — APPROVED** [review]

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**